resources/css/layout.css

```diff
@@ -37,16 +37,49 @@ body {
   }
 }*/
 
-body:not([wide]) .container {
-  max-width: theme('screens.lg');
-}
-
 header {
   overflow-x: hidden;
 }
 
 main {
   overflow-x: hidden;
+  padding: 0;
+  position: relative;
+  min-height: 450px;
+  display: grid;
+  column-gap: 1rem;
+}
+
+main.with-sidebar {
+  grid-template-columns: 1.329fr 0.671fr;
```

Comment:
Before I continue with the review, we should make a decision regarding this line. This is my proposed change, but I could be convinced it's not the right approach. I'm interested in your thoughts:

```suggestion
  grid-template-columns: 1.329fr 0.489fr;
```

Currently, 0.671fr adds about 100px of width to the sidebar.
0.489fr reduces this back down to 340px.

Arguments for 0.489fr:
* 340px has been fairly constrained for the sidebar in the past - maybe this is a good thing.
* We'll only need to optimize for MD and less on the sidebar.
* On XL, there won't be an "open invitation" to put primary content in the sidebar because of the extra new space.
* The sidebar on desktop will still flush neatly against the navbar and user card (though, the user card is going away before long).

Arguments for 0.671fr:
* 340px has been fairly constrained for the sidebar in the past - maybe this is _not_ a good thing.
* The main content area gets less space at XL, so it's probably less we need to fix.

I lean towards sticking with 0.489fr, but my confidence in this choice is not high and this will have design ramifications across the entire site for some time (hence why we should make a decision before moving on in this review).

IMO the 2nd column (sidebar) is much too at 0.671fr. This is harder to tell on dark mode, but easier to see on light mode:

![Screenshot 2023-09-03 at 9 56 03 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/95e3d604-f7c0-47a8-9314-9f76af3c01c3)

Setting this to 0.489fr gets us back to a crisp 340px width.

![Screenshot 2023-09-03 at 9 57 56 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/8c069176-d577-44cc-b39b-ba970e8b225b)

Let's land on a conclusion to this important decision first and then continue with the review.

Reply:
Thanks for bringing this up! I had very similar thoughts when browsing through the pages.

I'd agree and say let's stick with the 340px sidebar for now and concentrate on the main content area instead? You have some arguments for sticking with that for now.

https://github.com/RetroAchievements/RAWeb/pull/1821/commits/a0b455acfe6d88f6b069333ea3442002efc0a526

Reply:
We could give it (and the too card) 20-50px just let it breathe a bit.

I didn't find any major issue with giving everything more space in general. A lot has been prepared for this in the past so it doesn't completely fall apart when making it wider.

Reply:
Agreed, I just tested a few different numbers and 380px seems "just right".

Reply:
Perfect, let's go with that 👍

Reply:
Changed in https://github.com/RetroAchievements/RAWeb/pull/1821/commits/c6551d505fa25e5575f8768fec2b9072ec61ab7d